JADE.io (Australia).js

```diff
@@ -0,0 +1,819 @@
+		if (parsed && parsed.caseName) return parsed.caseName;
+		return textTrim(title.replace(/^\s*BarNet\s+Jade\s*-\s*/i, '').replace(/\s*-\s*BarNet\s+Jade.*$/i, ''));
+	}
+	return '';
+}
+
+// Case-name quality checks
+function isGenericCaseTitle(name) {
+	return /BarNet\s+Jade|Find\s+recent\s+Australian\s+legal|Judgments\s+And\s+Decisions\s+Enhanced/i.test(name || '');
+}
+
+// -------------------- Detection --------------------
+
+/**
+ * Zotero detection function - determines document type and whether page is scrapable
+ * Analyses URL patterns and title content to classify documents
+ *
+ * @param {Document} doc - HTML document object
+ * @param {string} url - Current page URL
+ * @returns {string|false} - "case", "statute", "multiple" or false if not scrapable
+ */
+function detectWeb(doc, url) {
+	// Multiples via JADE RSS/XML feeds
+	if (/\/xml\/.+\.xml$/i.test(url)) return "multiple";
+
+	// Content pages: articles, summary/outline views
+	if (/\/article\/\d+(?:\/section\/\d+)?(?:[?#].*)?$/i.test(url) || /\/summary\/mnc\//i.test(url) || /\/j\/\?a=outline&id=\d+/i.test(url)) {
+		const t = getTitle(doc) || "";
+		// If the title carries an MNC or looks like a case caption, treat as case
+		if (parseNeutralCitationFromTitle(t)) return "case";
+		if (/\bv\b/i.test(t)) return "case";
+		// Clear legislation signals → statute
+		if (looksLikeLegislationTitle(t) || /\/section\//i.test(url)) return "statute";
+		// Default JADE article pages to case if uncertain
+		if (/\/article\/\d+/.test(url)) return "case";
+		return "case";
+	}
+
+	// Broader fallback for any jade.io page that might contain legal content
+	if (url && url.includes('jade.io')) {
+		const t = getTitle(doc) || "";
+		if (parseNeutralCitationFromTitle(t)) return "case";
+		if (/\bv\b/i.test(t)) return "case";
+		if (looksLikeLegislationTitle(t)) return "statute";
+		// Check for legal content indicators
+		if (/\b(?:court|judgment|decision|case|act|regulation|statute)\b/i.test(t)) {
+			return /\b(?:act|regulation|statute)\b/i.test(t) ? "statute" : "case";
+		}
+	}
+
+	return false;
+}
+
+// -------------------- Multiples (RSS/XML) --------------------
+
+function getSearchResults(doc, checkOnly) {
+	let items = {};
+	let found = false;
+	const rows = doc.querySelectorAll("item");
+	for (const row of rows) {
+		const title = textTrim(ZU.xpathText(row, "title/text()"));
+		const href = textTrim(ZU.xpathText(row, "link/text()"));
+		if (!href || !title) continue;
+		if (checkOnly) return true;
+		found = true;
+		items[href] = title;
+	}
+	return found ? items : {};
+}
+
+// -------------------- doWeb (entry point) --------------------
+
+/**
+ * Main Zotero scraping function and entry point
+ * Routes to appropriate scraper based on document type detection
+ * Handles multiple document selection from search/feed pages
+ *
+ * @param {Document} doc - HTML document object
+ * @param {string} url - Current page URL
+ */
+function doWeb(doc, url) {
```

Comment:
Please use async functions and `await requestDocument()` instead of the deprecated, callback-based `processDocuments()`.

See the web translator template in Scaffold (Plus button -> Web Translator). No need to roll your own format for a method like this.